logger.cc

```diff
@@ -48,7 +49,7 @@ setident(const char* ident)
  *         -1 : message truncated (didn't fit into buffer)
  *         -2 : error returned by vsnprintf(3)
  */
-int
+void                                                  // REFACTOR: Changed return type from int to void to match header
```

Comment:
revert, the logmsg return type could be useful (even though we generally don't use it :sweat_smile: